tests/error-display/error-display.spec.tsx

```diff
@@ -123,6 +151,64 @@ describe("ErrorDisplay", () => {
                 ).toBe(error.description);
             }
         );
+
+        test.each(testData)(
+            "should render bookingsg %s error correctly",
+            (type: ErrorDisplayType) => {
+                render(
+                    <ThemeProvider theme={BookingSGTheme}>
+                        <ErrorDisplay type={type} />
+                    </ThemeProvider>
+                );
+
+                const error = errorDisplayData(type, "bookingsg");
+
+                expect(
+                    screen.getByRole("heading", {
+                        level: 1,
+                        name: error.title,
+                    })
+                ).toBeInTheDocument();
+
+                expect(screen.getByRole("img")).toHaveAttribute(
+                    "src",
+                    error.img.src
+                );
+
+                expect(
+                    screen.getByTestId(ERROR_DESCRIPTION_TEST_ID).textContent
+                ).toBe(error.description);
+            }
+        );
+
+        test.each(testData)(
+            "should render bookingsg with base illustration %s error correctly",
```

Comment:
I think a single scenario is sufficient, it's not necessary to test all variants for this prop

```suggestion
        test(
            "should use the specified illustration based on the illustrationScheme prop",
```